src/utiles/appx.ts

```diff
@@ -15,7 +15,7 @@ const LOCAL_CMS_PROVIDER = process.env.LOCAL_CMS_PROVIDER;
 export async function getPurchases(email: string): Promise<Course[]> {
   const value = await cache.get('courses', [email]);
   if (value) {
-    console.log(value)
+    console.log(value);
```

Comment:
remove this

Reply:
@siinghd i don't know why but yarn-lock file showing 12k+ changes